cluster/manifests/kubenurse/prometheus.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           value: "1"
       containers:
       - name: prometheus
-        image: container-registry.zalando.net/teapot/prometheus:v3.4.1-master-64
+        image: container-registry.zalando.net/teapot/prometheus:v3.4.1-master-66
```

Comment:
This is a regression because this image isn't a multi-arch image anymore: https://github.com/zalando-build/prometheus-pipeline/pull/67/files#r2224880520

Not sure how relevant it is for `kubenurse` but for the main prometheus it might be an issue.